modules/soc_http/main.sh

```diff
@@ -31,11 +39,44 @@ socDebugLog(){
 	fi
 }
 
-socDebugLog "Requesting SoC"
+getAndWriteSoc(){
+		echo 0 > $soctimerfile
+		socDebugLog "Requesting SoC"
+		echo 0 > $soctimerfile
+		soc=$(curl --connect-timeout 15 -s $ip | cut -f1 -d".")
+		if [ $soc -eq 0 ]; then
```

Comment:
Die Abfrage würde nur den SoC 0% als gültig ansehen. In der ursprünglichen Implementierung wurde der Exitcode `$?` eines Funktionsaufrufes ausgewertet. Hier jedoch wird der SoC direkt ausgegeben. Daher sollte die ursprüngliche Prüfung mit dem regulären Ausdruck verwendet werden.